.github/workflows/push-master.yml

```diff
@@ -17,21 +17,15 @@ on:
   push:
     branches:
       - documentation-test-branch
-
+permissions:
+  contents: write
 jobs:
   deploy_documentation:
     runs-on: ubuntu-latest
-    name: Build IBAMR and run tests
+    name: Compile and deploy doxygen documentation
     env:
-      # TODO: this is a copy-and-paste of the Travis version. Once this is
-      # working we should clean it up.
-      TRAVIS_BUILD_NUMBER: ${{ github.run_number }}
-      TRAVIS_COMMIT: ${{ github.sha }}
+      TRAVIS_BUILD_DIR: ${{ github.workspace }}
```

Comment:
Can we get rid of the rest of the travis variables at this point?

Reply:
we need to keep this variable! removing it is what caused the ibamr doc page to be blank